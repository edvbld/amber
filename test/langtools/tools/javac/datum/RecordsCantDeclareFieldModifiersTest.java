/*
 * @test /nodynamiccopyright/
 * @summary smoke negative test for datum classes
 * @compile/fail/ref=RecordsCantDeclareFieldModifiersTest.out -XDrawDiagnostics RecordsCantDeclareFieldModifiersTest.java
 */

public class RecordsCantDeclareFieldModifiersTest {
    record R(public int i);
}
