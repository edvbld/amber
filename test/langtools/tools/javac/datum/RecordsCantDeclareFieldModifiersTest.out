RecordsCantDeclareFieldModifiersTest.java:8:14: compiler.err.record.cant.declare.field.modifiers
1 error
